src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2698,6 +2698,9 @@ Right-click to remove the key combination, thereby deactivating the shortcut.</v
   <data name="VideoConference_DeprecationWarningButton.Content" xml:space="preserve">
     <value>Learn more</value>
   </data>
+  <data name="LearnMore.Content" xml:space="preserve">
```

Comment:
Can't we reuse uid's? Or is that against the idea of a unique identifier?

Reply:
Yeah, there are 2 different resources with same content ("Learn more"). It is Ok I think, maybe in the future someone wants to change just the one of them. Thanks!